### PR TITLE
typehead issue resolved

### LINKED ISSUE #80: Typeahead search should show selected state
It should be possible to see the selected values on the UI when picking from a typeahead list

### PR DESCRIPTION
**Pull request by anshul2401**
#80 resolved

### CODE REVIEW

File: lib/widgets/selectors.dart [hunk 1]
```diff
@@ -25,7 +35,9 @@ List<Widget> tenantSelector(BuildContext context, bool shouldShow,
           subtitle: Text(tenant.id),
         );
       },
-      onSuggestionSelected: onSuggestionSelected,
+      onSuggestionSelected: (suggestion) {
```
Comment (harsimranmaan) [inline lib/widgets/selectors.dart:38]: This would not work are the state is not set at all.

**Comment by harsimranmaan** [comment]
Does this work with saving? Also did you take a look at https://github.com/bigpanther/shipanther/commit/4214f3c56159da3a102309539605062187258a26?

**Comment by anshul2401** [comment]
> Does this work with saving? Also did you take a look at [4214f3c](https://github.com/bigpanther/shipanther/commit/4214f3c56159da3a102309539605062187258a26)?
I did some update, now its saving.

**Comment by harsimranmaan** [comment]
Thanks. After selecting the value, the user would see the value but it is hard to know which selector is which in case the screen has multiple selectors.

**Comment by harsimranmaan** [comment]
We need to handle the case of showing the initial value too but that would require more work and maybe a local cache to avoid the problem of n queries for each of the referenced fields. We can handle that separately.

File: lib/widgets/selectors.dart [hunk 2]
```diff
@@ -100,18 +135,28 @@ List<Widget> terminalSelector(BuildContext context, bool shouldShow,
...
   ];
 }
 
-List<Widget> orderSelector(BuildContext context, bool shouldShow,
-    void Function(Order) onSuggestionSelected) {
+List<Widget> orderSelector(
+    BuildContext context,
+    bool shouldShow,
+    void Function(Order) onSuggestionSelected,
+    TextEditingController _orderTypeAheadController) {
```
Comment (harsimranmaan) [inline lib/widgets/selectors.dart:150]: Don't user underscore for input fields. Also the _orderType is redundant and inconsistent.  See https://github.com/bigpanther/shipanther/commit/4214f3c56159da3a102309539605062187258a26#diff-8e5e037a8700213cd43e6734b09324f63a0c7bc4c47285374f8c89ede15afc4cR18 for reference

**Review by harsimranmaan — APPROVED** [review]

**Pull request merged**